osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -64,7 +84,7 @@ protected override double StrainValueOf(DifficultyHitObject current)
                 }
             }
 
-            return (1 + (speedBonus - 1) * 0.75) * angleBonus * (0.95 + speedBonus * Math.Pow(distance / single_spacing_threshold, 3.5)) / osuCurrent.StrainTime;
+            return (1 + (speedBonus - 1) * 0.75) * angleBonus * (0.95 + speedBonus * Math.Pow(distance / single_spacing_threshold, 3.5)) / Math.Max(deltaTime, 1);
```

Comment:
Can we split this giant line into something more digestible since we're changing it anyway?

Reply:
I'll see what I can do.